typescript/web/src/components/website/Navbar/Submenu.tsx

```diff
@@ -30,8 +30,8 @@ const DesktopSubmenu = (props: SubmenuProps) => {
       </NavLink.Desktop>
 
       <NavMenu {...getMenuProps()} animate={isOpen ? "open" : "closed"}>
-        <Box maxW="7xl" mx="auto" px="8">
-          <SimpleGrid spacing="10" columns={2}>
+        <Box maxW="7xl" mx="auto" px="400">
```

Comment:
400px as padding? What was you trying to do exactly?

Reply:
Not great I agree, I just trying to center the sub menu on the menu I click on but it is probably not the ideal solution